gamla/functional_generic.py

```diff
@@ -474,3 +474,23 @@ def countby_many(f):
             lambda x, y: x + 1 if x else 1,
         ),
     )
+
+
+def merge(*dicts):
```

Comment:
implement with `merge_with(last)`?

Reply:
Yes makes sense. Do we have a gamla implementation or do I use` toolz.last`?

Reply:
it's probably in the spreadsheet, but you can use `toolz.last` if it's not implemented yet